### PR TITLE
Added provider for videoconf persistent chat tests

### PR DESCRIPTION
**Pull request by pierre-lehnen-rc**

### CODE REVIEW

File: app.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
     "id": "bc4dd4a1-bf9b-408e-83a4-aba7eba0bf02",
-    "version": "0.1.0",
-    "requiredApiVersion": "^1.42.2",
+    "version": "0.1.1",
+    "requiredApiVersion": "^1.44.0",
```
Comment (tapiarafael) [inline app.json:4]: Do we have this version already? I get an error trying to install here.

File: videoConfProviders/PersistentChatVideoConfProvider.ts [hunk 2]
```diff
@@ -0,0 +1,37 @@
+/* eslint-disable @typescript-eslint/no-var-requires */
```
Comment (tapiarafael) [inline videoConfProviders/PersistentChatVideoConfProvider.ts:1]: Is this necessary?

**Review by tapiarafael — CHANGES_REQUESTED** [review]

File: videoConfProviders/PersistentChatVideoConfProvider.ts (hunk 2 above)
Comment (pierre-lehnen-rc) [reply videoConfProviders/PersistentChatVideoConfProvider.ts]: Not really, it came along as I copied another file; I've removed it now.

File: app.json (hunk 1 above)
Comment (pierre-lehnen-rc) [reply app.json:4]: It's the current alpha version.

**Review by tapiarafael — APPROVED** [review]

**Pull request merged**